src/server/debug.rs

```
@@ -809,36 +803,13 @@ impl<ER: RaftEngine> Debugger<ER> {
         Ok(())
     }
 
-    pub fn get_store_id(&self) -> Result<u64> {
-        let db = &self.engines.kv;
-        db.get_msg::<StoreIdent>(keys::STORE_IDENT_KEY)
-            .map_err(|e| box_err!(e))
-            .and_then(|ident| match ident {
-                Some(ident) => Ok(ident.get_store_id()),
-                None => Err(Error::NotFound("No store ident key".to_owned())),
-            })
-    }
-
-    pub fn get_store_info(&self) -> Result<DebugStoreInfo> {
+    pub fn get_store_ident(&self) -> Result<StoreIdent> {
         let db = &self.engines.kv;
         db.get_msg::<StoreIdent>(keys::STORE_IDENT_KEY)
             .map_err(|e| box_err!(e))
-            .and_then(|ident| match ident {
-                Some(ident) => Ok(DebugStoreInfo {
-                    store_id: ident.get_store_id(),
-                    api_version: ident.get_api_version(),
-                }),
-                None => Err(Error::NotFound("No store ident key".to_owned())),
-            })
-    }
-
-    pub fn get_cluster_id(&self) -> Result<u64> {
-        let db = &self.engines.kv;
-        db.get_msg::<StoreIdent>(keys::STORE_IDENT_KEY)
-            .map_err(|e| box_err!(e))
-            .and_then(|ident| match ident {
-                Some(ident) => Ok(ident.get_cluster_id()),
-                None => Err(Error::NotFound("No cluster ident key".to_owned())),
+            .map(|ident| match ident {
+                Some(ident) => ident,
+                None => StoreIdent::default(),
```

Should we throw an error here?